nile-backend/src/main/java/com/tp/Nile/models/Product.java

```diff
@@ -81,7 +73,7 @@
     @JsonManagedReference
     private Set<Question> questions = new HashSet<>();
 
-    public Product(Integer productId, Category categoryId, Vendor vendorId, Type type, Double price, String name, String description, String brand, Set<Photo> photos, Set<Feature> features, Set<Order> orders, Set<Question> questions) {
+    public Product(Integer productId, Category categoryId, Vendor vendorId, Type type, Double price, String name, String description, String brand, Set<Photo> photos, List<Feature> features, List<Order> orders, Set<Question> questions) {
```

Comment:
Constructors, getters, and setters aren't necessary with Lombak annotations

Reply:
I went ahead and deleted code related to getters/setters/constructors/hash/equals.